SingleSource/UnitTests/AArch64/acle-fmv-features.c

```diff
@@ -363,15 +363,23 @@ CHECK(mops, mops, mops, {
      );
 })
 
-static bool safe_try_feature(bool (*try_feature)(void)) {
+bool (*whitelist[])(void) = { try_ssbs2 };
```

Comment:
Having this separate list seems a bit roundabout, and I think it may fail to compile if `__HAVE_FUNCTION_MULTI_VERSIONING` is not defined? Might be better to have an extra argument for the check macro?

I'd also suggest to add a comment on why ssbs2 in particular is being whitelisted.